### PR TITLE
Not Updated for latest version

### LINKED ISSUE #1: Not Updated for latest version
The Base Adresses are not updated for version 421471 of mhw:I

### PR DESCRIPTION
**Pull request by Majwt**
Added new Pointers for the latest version of the game

### CODE REVIEW

**Comment by lyuha** [comment]
It's not work.

**Comment by Majwt** [comment]
Ok that's weird. It worked for me but I'll test again later.

**Comment by Majwt** [comment]
I found the problem! On line 39, `int questID : "MonsterHunterWorld.exe",0x0506F240,0x4C,0x14000000;`
The `0x14000000` should not be there, then it works.

Should i close this and make a new pull request or will you merge and remove the fault manually? I never really use Github and Git.

**Comment by lyuha** [comment]
Please update code and push again to your repository with same branch!
If you push [Majwt/issue1](https://github.com/Majwt/MHW-split/tree/Majwt/issue1) and it works, I will merge this PR!

Thanks to report issue and PR!

**Comment by Majwt** [comment]
Done!

File: monster-hunter-world.asl [hunk 1]
```diff
@@ -36,7 +36,7 @@ state("MonsterHunterWorld", "421471") {
     float sessionTime: "MonsterHunterWorld.exe",0x0506F240,0x90,0x46A0,0x14;
```
Comment (lyuha) [inline monster-hunter-world.asl]: Update this line!

Needs change sessionTime pointer.
Check This [CE table](https://opencheattables.org/viewtopic.php?f=12&t=20&p=31&hilit=monster+hunter+world#p31)

**Review by lyuha — CHANGES_REQUESTED** [review]

**Comment by Majwt** [comment]
Now it is done, I hope.

**Comment by lyuha** [comment]
It's works!
Thanks to PR!

**Pull request merged**